datasource/schema/list_attribute.go

```diff
@@ -44,6 +45,8 @@ var (
 type ListAttribute struct {
 	// ElementType is the type for all elements of the list. This field must be
 	// set.
+	//
+	// Element types that contain a dynamic type (i.e. types.Dynamic) are not supported.
```

Comment:
Should we provide the recommendation (😖 ) about what to do instead? e.g. If underlying dynamic values are required, replace this attribute definition with `DynamicAttribute` instead.

Reply:
I think your wording sounds good, I'm on board!

94d27c3 + b341f56⚡ - added to all the doc strings. I also added it to the blocks, although "replace this block definition with `DynamicAttribute`" is a loaded statement lol.... Wonder if I should just say nothing in those?

Reply:
All the error messages are also updated, this is done ⚡ 

0774ad3